engine.js

```diff
@@ -19,12 +21,33 @@ module.exports = function (options) {
   var types = options.types;
 
   var length = longest(Object.keys(types)).length + 1;
-  var choices = map(types, function (type, key) {
+  var typeChoices = map(types, function (type, key) {
     return {
       name: rightPad(key + ':', length) + ' ' + type.description,
       value: key
     };
   });
+  var scopePrompt = {
+    type: 'input',
+    name: 'scope',
+    message: 'What is the scope of this change (e.g. component or file name)? (press enter to skip)\n'
+  };
+  try {
+    var choices = JSON.parse(
```

Comment:
Can just use `require('./package.json')` instead of  `readFileSync` and `JSON.parse`.

Reply:
Definitely. I actually was aiming to use that, don't know why I derped.